pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -59,8 +59,9 @@ const (
 	loadbalancerDeleteFactor    = 1.2
 	loadbalancerDeleteSteps     = 13
 
-	activeStatus = "ACTIVE"
-	errorStatus  = "ERROR"
+	activeStatus  = "ACTIVE"
```

Comment:
```suggestion
	const activeStatus  = "ACTIVE"
```

Reply:
They should be within a const block already.